django/pyrog/api/serializers.py

```diff
@@ -3,15 +3,19 @@
 from pyrog import models
 
 
-class SourceSerializer(serializers.ModelSerializer):
+class ResourceSerializer(serializers.ModelSerializer):
+    attributes = serializers.PrimaryKeyRelatedField(many=True, read_only=True)
```

Comment:
I'm not sure this is something we always want to do (serialize a resource alongside its attributes)

Reply:
for instance @vmttn used the "full" query parameter to retrieve a source with its resources, attributes, input etc...

Reply:
Query parameters looks always hacky to me in a restful api especially used this way... And we're not serializing the full attributes contents here, only their ids.

Reply:
It's fine to include information about related resources, as long as they are not truly nested bc we would eventually have to manage circular dependencies.
Ids are ok.